client/my-sites/checkout/checkout/new-card-form.jsx

```diff
@@ -1,44 +1,48 @@
+import { isEmpty } from 'lodash';
+
 /**
  * External dependencies
  */
-import { isEmpty } from 'lodash';
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import PropTypes from 'prop-types';
+
```

Comment:
Is this space normal / auto-generated (codemod..?)?  Feels weirdish but frankly I'm still a lagger on style and other JS stuff.

Reply:
That was autogenerated indeed :)

Reply:
the extra newline between `PropTypes` and the next import is a byproduct of the PropType removal codemod and is a bug with the tooling.  I hope it doesn't catch on and become the new js style :)